<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stkPolarisTrackerWidget</class>
 <widget class="QWidget" name="stkPolarisTrackerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>stkPolarisTrackerWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>SerialPort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comPortChoiceComboBox">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>COM1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM8</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>ServerIP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="serverIPEdit">
       <property name="text">
        <string>localhost</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>ServerPort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="serverPortEdit">
       <property name="text">
        <string>18944</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connectServerButton">
       <property name="text">
        <string>Connect Server</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Tool1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>ROM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="romFileNameEdit">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>C:/config/PTool.rom</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Shift</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tool1_OffsetXEdit">
       <property name="text">
        <string>159.801</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tool1_OffsetYEdit">
       <property name="text">
        <string> -0.821</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tool1_OffsetZEdit">
       <property name="text">
        <string>21.357</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="openDeviceButton">
       <property name="text">
        <string>OpenDevice</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeDeviceButton">
       <property name="text">
        <string>CloseDevice</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startTrackingButton">
       <property name="text">
        <string>Start Tracking</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopTrackingButton">
       <property name="text">
        <string>Stop Tracking</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="toolsTableWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>56</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
